kalite/distributed/static/js/distributed/intro.js

```diff
@@ -0,0 +1,1263 @@
+/**
```

Comment:
Would be good to replace this (and the corresponding stylesheet) with a minified version.